xml/cl.xml

```diff
@@ -5718,6 +5736,28 @@ server's OpenCL/api-docs repository.
             <require comment="cl_channel_type">
                 <enum name="CL_UNORM_INT16"/>
                 <enum name="CL_FLOAT"/>
+        </extension>
+        <extension name="cl_khr_extended_versioning" supported="opencl">
+            <require comment="Types">
+                <type name="cl_version_khr"/>
+                <type name="cl_name_version_khr"/>
+            </require>
+            <require comment="Constants">
+                <enum name="CL_NAME_VERSION_MAX_NAME_SIZE_KHR"/>
```

Comment:
Since the `cl_name_version_khr` structure depends on the `CL_NAME_VERSION_MAX_NAME_SIZE_KHR` define I think it will make code generation simpler if this "Constants" section is swapped with the preceding "Types" section.

Reply:
Not sure we should be relying on that in code generation but I don't mind.